### PR TITLE
PAAS-278 Show an example YAML file for a project they can copy/paste

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/paas

### CODE REVIEW

**Comment by grosser** [comment]
screen shot 2016-07-21 at 1 10 05 pm

**Comment by grosser** [comment]
screen shot 2016-07-21 at 1 33 48 pm

File: plugins/kubernetes/app/views/kubernetes/roles/example.html.erb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      project: <%= project_label %>
+      role: app-server
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      name: <%= project_label %>-app-server
+      labels:
+        project: <%= project_label %>
+        role: app-server
```
Comment (jonmoter) [inline plugins/kubernetes/app/views/kubernetes/roles/example.html.erb]: Could you add some example annotations here for secrets, with a comment on how they can be used?

**Comment by jonmoter** [comment]
Thanks!  👍

**Pull request merged**